Homework4/ArrFunc_TS6.cpp

```diff
@@ -0,0 +1,66 @@
+#include "ArrFunc_TS6.hpp"
+
+#include <fstream>
+#include <iostream>
+#include <random>
+
+// RANDOM INIT ARRAY
+void rInitArray(int *const array_begin, int const *const array_end) {
+  if (array_begin == nullptr || array_end == nullptr ||
+      array_begin >= array_end) {
+    return;  // error
+  }
+  std::random_device rd;
+  std::mt19937 mt(rd());
+  std::uniform_int_distribution<int> GenInt(k_min_random_value,
+                                            k_max_random_value);
+  for (int *index = array_begin; index != array_end; index++) {
+    *index = GenInt(mt);
+  }
+  // ok
+}
+
+void fInitArray(int *const array_begin, int const *const array_end) {
+  if (array_begin == nullptr || array_end == nullptr ||
+      array_begin >= array_end) {
+    return;  // error
+  }
+  std::ifstream InitFile;
+  InitFile.open(k_file_name);
```

Comment:
зачем после создания объекта `ifstream` писать функцию open?
вы же изучали конструктора, так вот если передать имя файла в конструктор, то он там будет открывать файл
т.е.
```suggestion
  std::ifstream InitFile{k_file_name};
```
так намного эфективней